core/db/migrate/20210722090705_add_store_id_to_spree_customer_returns.rb

```diff
@@ -4,7 +4,7 @@ def change
       add_column :spree_customer_returns, :store_id, :bigint
       add_index :spree_customer_returns, :store_id
       Spree::CustomerReturn.reset_column_information
-      Spree::CustomerReturn.find_each { |cr| cr.update_column(:store_id, cr.order.store_id) }
+      Spree::CustomerReturn.find_each { |cr| cr.update_column(:store_id, cr.order&.store_id || Spree::Store.default.id) }
```

Comment:
```suggestion
      Spree::CustomerReturn.find_each { |cr| cr.update_column(:store_id, cr.order&.store_id || default_store_id) }
```

Reply:
@damianlegawiec Right, thanks for spotting that :)